includes/site-health/audit-autoloaded-options/hooks.php

```diff
@@ -42,15 +42,15 @@ function perflab_aao_register_admin_actions() {
  *
  * @since 3.0.0
  */
-function perflab_aao_handle_update_autoload() {
+function perflab_aao_handle_update_autoload(): void {
 	check_admin_referer( 'perflab_aao_update_autoload' );
 
 	if ( ! isset( $_GET['option_name'], $_GET['autoload'] ) ) {
 		wp_die( esc_html__( 'Missing required parameter.', 'performance-lab' ) );
 	}
 
 	$option_name = sanitize_text_field( wp_unslash( $_GET['option_name'] ) );
-	$autoload    = isset( $_GET['autoload'] ) ? rest_sanitize_boolean( $_GET['autoload'] ) : false;
+	$autoload    = rest_sanitize_boolean( $_GET['autoload'] );
```

Comment:
Note the removal of the `isset( $_GET['autoload'] )` check since it is already checked above.